src/main/java/com/github/havardh/javaflow/ast/Class.java

```diff
@@ -26,13 +27,22 @@
    * @param fields the list of member fields
    * @param getters the list of getters in the class
    */
-  public Class(CanonicalName name, Parent parent, List<Field> fields, List<Method> getters) {
+  public Class(CanonicalName name, Parent parent, List<Type> types, List<Field> fields, List<Method> getters) {
```

Comment:
`types` is not that self explanatory in this context, lets call it something like `innerClasses`.